docs/development.rst

```diff
@@ -37,6 +31,8 @@ During development, once Girder is started via ``python -m girder``, the server
 will reload itself whenever a Python file is modified.
 
 To get the same auto-building behavior for JavaScript, we use ``grunt-watch``.
+Since ``grunt`` is installed in the ``./node_modules/.bin/`` directory, you could
+conveniently update the ``PATH`` running ``export PATH=$(pwd)/node_modules/.bin:$PATH``.
```

Comment:
Seems more complete to add "You may alternately install `grunt` globally as described in [Grunt's documentation](http://gruntjs.com/getting-started) to avoid setting your `PATH`, though this may require sudo privileges."